### PR TITLE
feat(celery): add celery beat for scheduled tasks

### PR DESCRIPTION
**Pull request by jonathaningram**
We're not currently running any scheduled tasks. This change adds the
celery beat application to our build.

### CODE REVIEW

File: api/manifest-dev.yml [hunk 1]
```diff
@@ -29,3 +29,9 @@ applications:
     health-check-type: process
     no-route: true
     command: make run-celery
+  - name: notify-celery-beat-((stg))
+    <<: *defaults
+    memory: 256M
+    health-check-type: process
+    no-route: true
+    command: make run-celery
```
Comment (andrewoh531) [inline api/manifest-dev.yml]: `command: make run-celery-beat`?

**Comment by jonathaningram** [comment]
@andrewoh531 can you please review again? We should both do actual tests on the build for this one. Also before I approve and merge to master we should test stg.

File: api/app/celery/research_mode_tasks.py [hunk 2]
```diff
@@ -11,34 +11,26 @@
 from app.config import QueueNames
 from app.celery.process_ses_receipts_tasks import process_ses_results
 
-temp_fail = "7700900003"
-perm_fail = "7700900002"
-delivered = "7700900001"
+temp_fail = "409000003"
```
Comment (andrewoh531) [inline api/app/celery/research_mode_tasks.py]: Do you know what research mode is? I can see it's one of the modes a service can be placed under but there isn't documentation around it I could find.

Comment (jonathaningram) [reply api/app/celery/research_mode_tasks.py]: `research_mode` is a property on a service that can be enabled by platform admin.

**Pull request merged**